### PR TITLE
Update 2023

### PR DESCRIPTION
**Pull request by keatincf**
Update the project to the latest publicly released version

### CODE REVIEW

**Comment by keatincf** [comment]
The README.md should probably be updated with the latest public release case #.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,52 @@
+## [0.13.2](https://github.com/act3-ace/safe-autonomy-dynamics/compare/v0.13.1...v0.13.2) (2023-09-21)
```
Comment (jamie-cunningham) [inline CHANGELOG.md]: Will these tags be valid?

File: README.md [hunk 2]
```diff
@@ -1,63 +1,104 @@
...
 ## Public Release
-Distribution A. Approved for public release; distribution is unlimited. Case Number: AFRL-2022-3202
+
+Approved for public release; distribution is unlimited. Case Number: AFRL-2023-6155
+
+A prior version of this repository was approved for public release. Case Number: AFRL-2022-3202
 
 ## Team
+
 Umberto Ravaioli,
```
Comment (jamie-cunningham) [inline README.md:102]: Charles should be added to this list

File: requirements.dep.txt
Comment (jamie-cunningham) [inline requirements.dep.txt:1]: Is this file still needed?

File: CHANGELOG.md (hunk 1 above)
Comment (keatincf) [reply CHANGELOG.md]: They will not be.  The scrubbing of the repo caused the change in the URLs to something that doesn't exist.  If they weren't changed, they would have pointed to something the public can't access.  We could either remove the links from the change log, with future releases adding valid links to new changes, or we could remove the log completely.  Then, we'd start with a fresh change log, but only containing information for the new changes.  I'd probably lean toward removing the links as I could see someone wanting to know what has changed since the last public release.

File: requirements.dep.txt
Comment (keatincf) [reply requirements.dep.txt:1]: I'm going to say we don't.  It's a file created by the release process, but I'm not sure what it's used for.  poetry.lock contains the same information on the exact dependency versions used.  I don't believe it's included in the packages that are created for a release.  So, we should be able to remove this file and update release.sh to not generate the file.

File: CHANGELOG.md (hunk 1 above)
Comment (jamie-cunningham) [reply CHANGELOG.md]: Yes I agree we definitely should keep the changelog. I'm in favor of removing the links completely.

Comment (jamie-cunningham) [reply CHANGELOG.md]: It looks like those are valid urls as far as GitHub compare is concerned, the tags just don't exist. I don't think it's worth trying to keep the tags synced just to make the links work.

Comment (keatincf) [reply CHANGELOG.md]: Links were removed.

**Comment by keatincf** [comment]
Rebased the branch on the latest main and made adjustments to the workflow.  I also kept CHANGELOG.md the same as from the 1.1.0 release, as it wouldn't make sense to have 0.x release info added in after the 1.1.0 release.  The CHANGELOG.md should be updated with the 1.2.0 release.

I also made an adjustment to the Github release so that the title of the release is the version rather than the default behavior of using the commit message.

**Review by jamie-cunningham — APPROVED** [review]
lgtm
